diff --git a/kani-driver/src/frontend/schema_utils.rs b/kani-driver/src/frontend/schema_utils.rs
index 882961ca9..6006222f9 100644
--- a/kani-driver/src/frontend/schema_utils.rs
+++ b/kani-driver/src/frontend/schema_utils.rs
@@ -936,7 +936,7 @@ fn from_harness_result(
             is_automatically_generated: harness.is_automatically_generated,
             has_loop_contracts: harness.has_loop_contracts,
             is_bounded: harness.is_bounded,
-            attributes: harness.attributes.clone(),
+            attributes: HarnessAttributes::new(kani_metadata::HarnessKind::Proof),
             outcome,
             resolved_solver,
             resolved_unwind,
