diff --git a/kani-driver/src/main.rs b/kani-driver/src/main.rs
index 85df26e85..d630e4459 100644
--- a/kani-driver/src/main.rs
+++ b/kani-driver/src/main.rs
@@ -169,6 +169,9 @@ fn verify_project(project: Project, session: KaniSession) -> Result<()> {
     let started_at = OffsetDateTime::now_utc();
     let start_instant = Instant::now();
 
+    if let Some(path) = &session.args.export_json {
+        let _ = std::fs::remove_file(path);
+    }
     let harnesses = session.determine_targets(project.get_all_harnesses())?;
     debug!(n = harnesses.len(), ?harnesses, "verify_project");
 
