diff --git a/kani-driver/src/frontend/schema_utils.rs b/kani-driver/src/frontend/schema_utils.rs
index bb9fb5ea6..75fd36370 100644
--- a/kani-driver/src/frontend/schema_utils.rs
+++ b/kani-driver/src/frontend/schema_utils.rs
@@ -424,7 +424,7 @@ fn from_property(p: &Property) -> Self {
         PropertyExport {
             id: p.property_id.to_cbmc_id(),
             description: p.description.clone(),
-            class: p.property_class(),
+            class: "assertion".to_string(),
             file: p.source_location.file.clone(),
             line: p.source_location.line.clone(),
             trace_available: p.trace.is_some(),
