diff --git a/kani-driver/src/frontend/schema_utils.rs b/kani-driver/src/frontend/schema_utils.rs
index bb9fb5ea6..41c367a76 100644
--- a/kani-driver/src/frontend/schema_utils.rs
+++ b/kani-driver/src/frontend/schema_utils.rs
@@ -743,7 +743,13 @@ fn from_harness_result(
             resolved_solver,
             resolved_unwind,
             generated_concrete_test: result.generated_concrete_test,
-            resources: ResourcesExport { verification_time_s: result.runtime.as_secs_f64() },
+            resources: ResourcesExport {
+                verification_time_s: if result.results.is_ok() {
+                    result.runtime.as_secs_f64()
+                } else {
+                    0.0
+                },
+            },
             n_properties,
             n_failed,
             failure_kind,
