diff --git a/kani-driver/src/frontend/schema_utils.rs b/kani-driver/src/frontend/schema_utils.rs
index bb9fb5ea6..5887946f3 100644
--- a/kani-driver/src/frontend/schema_utils.rs
+++ b/kani-driver/src/frontend/schema_utils.rs
@@ -111,7 +111,7 @@ fn build_run_context(
 
 fn checks_flags_from(args: &VerificationArgs) -> ChecksFlags {
     ChecksFlags {
-        memory_safety: args.checks.memory_safety_on(),
+        memory_safety: true,
         overflow: args.checks.overflow_on(),
         unwinding: args.checks.unwinding_on(),
         undefined_function: args.checks.undefined_function_on(),
