diff --git a/kani_metadata/src/harness.rs b/kani_metadata/src/harness.rs
index 4d75ebed9..054e8f63a 100644
--- a/kani_metadata/src/harness.rs
+++ b/kani_metadata/src/harness.rs
@@ -72,6 +72,7 @@ pub struct HarnessAttributes {
 }
 
 #[derive(Clone, Eq, PartialEq, Debug, Display, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
 pub enum HarnessKind {
     /// Function was annotated with `#[kani::proof]`.
     #[strum(serialize = "#[kani::proof]")]
