diff --git a/kani-driver/src/frontend/schema_utils.rs b/kani-driver/src/frontend/schema_utils.rs
index bb9fb5ea6..0f1c45920 100644
--- a/kani-driver/src/frontend/schema_utils.rs
+++ b/kani-driver/src/frontend/schema_utils.rs
@@ -288,6 +288,7 @@ struct ExportedRun {
     tools: ToolsExport,
     enabled_unstable_features: Vec<String>,
     harness_selection: HarnessSelectionExport,
+    #[serde(skip_serializing)]
     harness_timeout_s: Option<f64>,
     configuration: ConfigurationExport,
     outcome: RunOutcome,
